frontend/src/store/index.js

```diff
@@ -1016,6 +1016,9 @@ const getters = {
   canDeleteShoots (state) {
     return canI(state.subjectRules, 'delete', 'core.gardener.cloud', 'shoots')
   },
+  canPatchShootBindings (state) {
+    return canI(state.subjectRules, 'patch', 'core.gardener.cloud', 'shoots/binding')
```

Comment:
No. I still do not understand why patch is supported.
https://kubernetes.io/docs/reference/access-authn-authz/authorization/#determine-the-request-verb

Reply:
we clarified this thanks @rfranzke